terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.test.ts

```diff
@@ -710,6 +710,7 @@ runner_types:
         instance_type: 'c5.2xlarge',
         os: 'linux',
         max_available: 1,
+        min_available: 10,
```

Comment:
Looks really weird to see min_available be larger than max_available, even if this is a test